nix/scripts/purge.sh

```diff
@@ -22,12 +23,10 @@ nix_purge_linux_multi_user_users() {
 }
 
 nix_purge_darwin_multi_user_service() {
-    cd /Library/LaunchDaemons
-    NIX_SERVICES=(org.nixos.darwin-store.plist org.nixos.nix-daemon.plist)
-    for NIX_SERVICE in "${NIX_SERVICES[@]}"; do
-        sudo launchctl unload "${NIX_SERVICE}" || true
-        sudo launchctl remove "${NIX_SERVICE}" || true
-    done
+    sudo launchctl unload /Library/LaunchDaemons/org.nixos.nix-daemon.plist
```

Comment:
I don't mind the removal of the loop, though I wonder why you did it.

Reply:
initially it was just an experiment to see if using this script from https://nixos.org/manual/nix/stable/installation/uninstall.html#:~:text=Stop%20and%20remove%20the%20Nix%20daemon%20services%3A changes the outcome or not and turns out it did.

there was something different about old approach 
i.e cd into the Library/LaunchDaemons folder and then execute unload vs passing full path to unload but yeah since it worked better i kept it this way.

Reply:
If the `cd` was the issue then the loop could still stay, but it doesn't really matter.